syntax = "proto3";

package orders;

service OrderService {
  rpc SendMessage (MessageRequest) returns (MessageResponse);
}

message MessageRequest {
  string message = 1;
}

message MessageResponse {
  string response = 1;
}
